cfgov/unprocessed/js/modules/util/dom-traverse.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+require( '../polyfill/class-list' );
+
+/**
+ * Get the sibling nodes of a dom node.
+ *
+ * @param {HTMLNode} elem - DOM element to get siblings of.
+ * @param {string} className - CSS classname.
+ * @returns {Array} List of sibling nodes.
+ */
+function getSiblings( elem, className ) {
+  var siblings = [];
+  var sibling = elem.parentNode.firstChild;
+  for ( ; sibling; sibling = sibling.nextSibling ) {
+    // Check that sibling is an HTMLNode,
+    // is not the original element,
+    // and contains the CSS class we're looking for.
+    if ( sibling.nodeType === 1 &&
+         sibling !== elem &&
+         sibling.classList.contains( className ) ) {
+      siblings[siblings.length] = sibling;
+    }
+  }
+  return siblings;
+}
+
+/**
+ * Get the sibling nodes of a dom node.
+ *
+ * @param {NodeList} elems - List of DOM elements.
+ * @param {HTMLNode} exclude - DOM element to exlude from elems.
+ * @returns {Array} edited elems list or original elems list,
+ *   if exlude was not found.
+ */
+function not( elems, exclude ) {
+  var elemsArr = Array.prototype.slice.call( elems );
+  var index = elemsArr.indexOf( exclude );
+  if ( index > -1 ) {
+    return elemsArr.splice( index, 1 );
+  }
+  return elemsArr;
+}
+
+/**
+ * Get the sibling nodes of a dom node.
+ *
+ * @param {HTMLNode} elem - List of DOM elements.
```

Comment:
Fixed in, though parameters looked right? https://github.com/cfpb/cfgov-refresh/pull/1290